src/test/txpackage_tests.cpp

```diff
@@ -122,7 +122,7 @@ BOOST_FIXTURE_TEST_CASE(package_validation_tests, TestChain100Setup)
 
     // A single, giant transaction submitted through ProcessNewPackage fails on single tx policy.
     CTransactionRef giant_ptx = create_placeholder_tx(999, 999);
-    BOOST_CHECK(GetVirtualTransactionSize(*giant_ptx) > MAX_PACKAGE_SIZE * 1000);
+    BOOST_CHECK(GetVirtualTransactionSize(*giant_ptx) > DEFAULT_ANCESTOR_SIZE_LIMIT_KVB * 1000);
```

Comment:
nit 0774c39e86ca52b747ace02f367617f4d6d88305

should this be checking against `MAX_PACKAGE_KWU`?

Reply:
> fails on single tx policy

it's a single tx test check, not a package limit check? :shrug: